README.md

```diff
@@ -26,20 +46,25 @@ Additional description about the project and its features...
 
 #### Click on the button below..
 
-[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/import/git?s=https%3A%2F%2Fgithub.com%2Fvercel%2Fnext.js%2Ftree%2Fcanary%2Fexamples%2Fhello-world)
+[![Run on Repl.it](https://repl.it/badge/github/DcRonan/Tic-Tac-Toe)](https://repl.it/github/DcRonan/Tic-Tac-Toe)
 
-## Run Tests
+## Getting Started
 
 ### Install
+**To get this working on your local computer, follow the steps below!**
+
 ```
-Download code
+1. Install Ruby
+2. Open the Terminal
+3. Run 'bundle install'
+4. Download code by entering 'git clone git@github.com:DcRonan/Tic-Tac-Toe.git' in the terminal
```

Comment:
Don't you think that that 4-th step comes before the 3-rd step? 
I mean we download the code, `cd` into the folder and then run `bundle install`.

Reply:
Also, it's preferable to use the `http` download method, I mean 
`https://github.com/DcRonan/Tic-Tac-Toe.git`

Suggest the **ssh** method to people allowed to change the code